agent/api/task/task.go

```diff
@@ -3019,7 +3019,7 @@ func (task *Task) GetCredentialSpecResource() ([]taskresource.TaskResource, bool
 }
 
 // getAllCredentialSpecRequirements is used to build all the credential spec requirements for the task
```

Comment:
super nit:  comment also needs to be update to `GetAllCredentialSpecRequirements`.